src/Authenticators/SAMLLoginHandler.php

```diff
@@ -2,10 +2,68 @@
 
 namespace SilverStripe\SAML\Authenticators;
 
-use SilverStripe\Security\MemberAuthenticator\LoginHandler;
+use SilverStripe\Control\Controller;
+use SilverStripe\Control\HTTPRequest;
+use SilverStripe\Control\HTTPResponse;
+use SilverStripe\Control\HTTPResponse_Exception;
+use SilverStripe\Control\RequestHandler;
+use SilverStripe\SAML\Helpers\SAMLHelper;
 
-class SAMLLoginHandler extends LoginHandler
+class SAMLLoginHandler extends RequestHandler
 {
+    private static $url_handlers = [
+        '' => 'login',
+    ];
+
+    private static $allowed_actions = [
+        'login',
+        'LoginForm'
+    ];
+
+    private static $dependencies = [
+        'helper' => '%$' . SAMLHelper::class
```

Comment:
We probably need a public set/getHelper for this too

Reply:
Have done this now :)